src/jquery.tabletojson.js

```diff
@@ -102,9 +102,19 @@
 
           $row = $(row);
 
-          var isEmpty = ($row.find('td').length === $row.find('td:empty').length) ? true : false;
+          var isEmptyRow = function (row){
+            let result = true;
```

Comment:
Please use `var` over `let` here to preserve compatibility. The build doesn't do any ESX conversions.